alt_e2eshark/run.py

```diff
@@ -228,6 +241,13 @@ def run_tests(
                 outputs_raw = config.run(compiled_artifact, inputs, func_name=func_name)
                 outputs_raw.save_to(log_dir + "output")
 
+            # benchmark inference time with compiled module
+            curr_stage = "benchmark"
+            if curr_stage in stages:
+                notify_stage()
+                # TODO: make repetitions configurable from a command line arg
+                mean_time_ms = config.benchmark(compiled_artifact, inputs, repetitions=5, func_name=func_name)
```

Comment:
In interest of nightly times, maybe drop to 2 or 3 repetitions?

Reply:
I originally had it at ten then moved to five haha. I'll drop the default to 3.